reports/hemang.md

```diff
@@ -0,0 +1,228 @@
+---
+layout: report
+permalink: /GSoC/2017/ProjectReport/HemangKumar
+student: Hemang Kumar
+organisation: coala
```

Comment:
Make it default to coala, i.e. if not overriden?

Reply:
Yeah I think its better to have it configurable. Making it a default will require adding it in markup in _layout/reports.md file as I understand. But it'll be good to have just one configuration file (that is this file) and only file governing the layout  (_layouts/report.md , which just has the design and no config defaults)